docs/cub/benchmarking.rst

```diff
@@ -288,3 +288,41 @@ For example:
         "speedup": 1
       }
     ]
+
+
+Profiling benchmarks with Nsight Compute
+--------------------------------------------------------------------------------
+
+If you want to see profiling metrics on source code level,
+you have to recompile your benchmarks with the `-lineinfo` option.
+With cmake, you can just add `-DCMAKE_CUDA_FLAGS=-lineinfo` when invoking cmake in the `build` directory:
+
+.. code-block:: bash
+
+    cmake .. --preset=cub-benchmark -DCMAKE_CUDA_FLAGS=-lineinfo -DCMAKE_CUDA_ARCHITECTURES=90 # TODO: Set your GPU architecture
```

Comment:
Is the todo meant to stay ;)

Reply:
Yes it is. The user needs to make sure to set this value according to their system.